.php-cs-fixer.cache

```diff
@@ -0,0 +1 @@
case_semicolon_to_colon":true,"switch_case_space":true,"visibility_required":{"elements":["method","property"]},"encoding":true,"full_opening_tag":true},"hashes":{"\/private\/var\/folders\/tx\/nrprq6gj0m14sk83c2n4tv_m0000gn\/T\/PHP CS Fixertemp_folder13927\/src\/Entity\/Comment\/Comment.php":1086955853,"\/private\/var\/folders\/tx\/nrprq6gj0m14sk83c2n4tv_m0000gn\/T\/PHP CS Fixertemp_folder16180\/src\/Repository\/UserRepository.php":1785589915,"\/private\/var\/folders\/tx\/nrprq6gj0m14sk83c2n4tv_m0000gn\/T\/PHP CS Fixertemp_folder6524\/src\/Entity\/Post\/Post.php":3174746701,"\/private\/var\/folders\/tx\/nrprq6gj0m14sk83c2n4tv_m0000gn\/T\/PHP CS Fixertemp_folder10520\/src\/Entity\/User\/User.php":2634575455,"\/private\/var\/folders\/tx\/nrprq6gj0m14sk83c2n4tv_m0000gn\/T\/PHP CS Fixertemp_folder6801\/src\/Repository\/CommentRepository.php":2506903177,"\/private\/var\/folders\/tx\/nrprq6gj0m14sk83c2n4tv_m0000gn\/T\/PHP CS Fixertemp_folder971\/src\/Repository\/PostRepository.php":240198322,"\/private\/var\/folders\/tx\/nrprq6gj0m14sk83c2n4tv_m0000gn\/T\/PHP CS Fixertemp_folder5286\/src\/Controllers\/Home\/HomeController.php":3158579141,"\/private\/var\/folders\/tx\/nrprq6gj0m14sk83c2n4tv_m0000gn\/T\/PHP CS Fixertemp_folder3775\/src\/Entity\/EntityInterface.php":3718657481,"\/private\/var\/folders\/tx\/nrprq6gj0m14sk83c2n4tv_m0000gn\/T\/PHP CS Fixertemp_folder16413\/src\/Repository\/Repository.php":1342896778,"\/private\/var\/folders\/tx\/nrprq6gj0m14sk83c2n4tv_m0000gn\/T\/PHP CS Fixertemp_folder6653\/src\/Repository\/CommentRepository.php":2506903177,"\/private\/var\/folders\/tx\/nrprq6gj0m14sk83c2n4tv_m0000gn\/T\/PHP CS Fixertemp_folder9414\/src\/Repository\/PostRepository.php":240198322,"\/private\/var\/folders\/tx\/nrprq6gj0m14sk83c2n4tv_m0000gn\/T\/PHP CS Fixertemp_folder8883\/src\/Repository\/Repository.php":1342896778,"\/private\/var\/folders\/tx\/nrprq6gj0m14sk83c2n4tv_m0000gn\/T\/PHP CS Fixertemp_folder1098\/src\/Repository\/Repository.php":4086487381,"\/private\/var\/folders\/tx\/nrprq6gj0m14sk83c2n4tv_m0000gn\/T\/PHP CS Fixertemp_folder6709\/src\/Repository\/Repository.php":4086487381,"\/private\/var\/folders\/tx\/nrprq6gj0m14sk83c2n4tv_m0000gn\/T\/PHP CS Fixertemp_folder8351\/src\/Repository\/Repository.php":1433466543,"\/private\/var\/folders\/tx\/nrprq6gj0m14sk83c2n4tv_m0000gn\/T\/PHP CS Fixertemp_folder16345\/src\/Repository\/Repository.php":1433466543,"\/private\/var\/folders\/tx\/nrprq6gj0m14sk83c2n4tv_m0000gn\/T\/PHP CS Fixertemp_folder5225\/src\/Repository\/Repository.php":292391938,"\/private\/var\/folders\/tx\/nrprq6gj0m14sk83c2n4tv_m0000gn\/T\/PHP CS Fixertemp_folder14346\/src\/Repository\/Repository.php":292391938,"src\/Repository\/Repository.php":292391938,"\/private\/var\/folders\/tx\/nrprq6gj0m14sk83c2n4tv_m0000gn\/T\/PHP CS Fixertemp_folder14670\/src\/Repository\/Repository.php":292391938,"\/private\/var\/folders\/tx\/nrprq6gj0m14sk83c2n4tv_m0000gn\/T\/PHP CS Fixertemp_folder8993\/src\/Repository\/Repository.php":292391938}}
```

Comment:
this file should be in your gitignore ^^, (it's a cache file to speed up cs-fixer ;) )